auth-web/src/components/auth/IncorpOrRegisterInfo.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <v-container id="incorporate-info-container">
+    <v-row>
+      <!-- Info Column -->
+      <v-col cols="12" md="6">
+        <h2>Incorporate or Register</h2>
+        <v-list-item class="list-item">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              If you have an approved Name Request (NR number), or you want a <NumberedCompanyTooltip />,
```

Comment:
https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended
I am not sure ;should it be written `<NumberedCompanyTooltip /> `OR` <NumberedCompanyTooltip ></NumberedCompanyTooltip>`

Reply:
I think in this instance, since we are using the single file component `App` structure, it is recommending to go with the self closing tag (I think :P).

Reply:
I thought we shouldn't use self closing tags within html templates.. either way , i am good

Reply:
Should it be `<numbered-company-tooltip />`?

Reply:
This isn't a DOM template. It's a Single File Component (SFC) :)

Reply:
> Should it be `<numbered-company-tooltip />`?

Good question, i'm not 100% sure but from the recommendations it specifies `<ExampleComponent />` for SFC or `<example-component></exmaple-component>` for everywhere. So i'm guessing the latter is for more generic use, but i believe both are valid.